vivisect/tools/graphutil.py

```diff
@@ -629,7 +629,16 @@ def clearGraphRouting(graph):
     for nid, ninfo in graph.getNodes():
         graph.getNodeProps(nid)['up'  ] = False
         graph.getNodeProps(nid)['down'] = False
```

Comment:
currently (to my knowledge) there is currently no other way to do this in the api